tests/fixtures/optimizer/simplify.sql

```diff
@@ -625,22 +625,22 @@ t0.x = t1.x AND t0.y < t1.y AND t0.y <= t1.y;
 t0.x = t1.x AND t0.y < t1.y AND t0.y <= t1.y;
 
 --------------------------------------
--- Coalesce
+-- COALESCE
 --------------------------------------
 COALESCE(x);
 x;
 
 COALESCE(x, 1) = 2;
-x = 2 AND NOT x IS NULL;
+x = 2;
```

Comment:
I just realized this is incorrect because if `x` is `NULL` it'll result in `NULL` whereas before it'd result in `FALSE`. I'll try to fix it soon.

Reply:
Tried to address this in https://github.com/tobymao/sqlglot/pull/2386/commits/b5d0c7f5e0fcaf6fece342c7cdbd201563184fb1, need to think about it a bit more though probably..